internal/config/annotations.go

```diff
@@ -45,6 +45,12 @@ const (
 	ANNOTATION_REPO_PATH = "repo-path"
 	// ANNOTATION_PG_PORT is for storing the PostgreSQL port for a cluster
 	ANNOTATION_PG_PORT = "pg-port"
+	// ANNOTATION_GCS_BUCKET is for storing the name of the GCS bucket used by
```

Comment:
:question: Do we also want to track `gcs-key-type`?  I'm not sure if there is any practical reason this would ever be anything but the default of `service` out in the wild (the pgBackRest docs mention that `token` is for local testing, so thinking not...), but if so, we would also want to track which type of key is in the secret to know how to use it in a subsequent restore.

Reply:
Yup, that's why I didn't add it, but it seems like it's fairly easy to add, so...may as well.